STAT302 - assignment 2 Q 3.2

In [1]:
# Q 3.2 a
caloriesHat = 109.3 + 1 * 11 - 3.7 * 1

The caloriesHat is 116.6

In [2]:
# Q 3.2 b
110 - 116.6

[1] -6.6

Frosted Flake, his calorie content was 6.60 calories less than the model predicted.